### ISSUE TITLE
Please consider this android client for wiki

### ISSUE THREAD

**Issue by stefan-niedermann** [owncloud/notes#156]
https://github.com/stefan-niedermann/OwnCloud-Notes

It is still early beta, but maybe some people want to test it and it is worth about mention in the [Wiki](https://github.com/owncloud/notes/wiki)?

cc @jancborchardt @owncloud/designers

**Comment by Henni** [comment]
@stefan-niedermann awesome! Great to see this.
I added your app to the wiki.
Did you already release it to some app store (Google Play or F-Droid)?

**Comment by stefan-niedermann** [comment]
My original plan was to provide it for free on F-Droid and for some money on Google Play (for people who want to donate). But Google Play requires a Credit Card (MasterCard, ...) which i don't have. So i'm not able to provide it on Google Play.
F-Droid is still planned, but i didn't ever publish an app on F-Droid so i have to do some research what i have to do.

TL;DR: Not yet.

**Comment by Henni** [comment]
@stefan-niederman Please keep me updated.

**Comment by jancborchardt** [comment]
@stefan-niedermann it would be so great finally having a good Notes app for Android! :) Let me know as soon as it’s on F-Droid (it’s super simple, you just add a txt file to their repo last I added an app). Important is that it should properly work offline. :)

Maybe also link up with the folks of https://github.com/aykit/MyOwnNotes: @steppenhahn @ramschmaerchen & @maltefiala, who unfortunately haven’t found time recently to continue on their app.

Also cc @davivel @masensio @tobiasKaminsky @AndyScherzinger @stoyicker @kernald from our Android app folks, and @David-Development from the [News Android app](https://github.com/owncloud/news-android-app) in case they want to check it out, review the code or contribute. :)

**Comment by AndyScherzinger** [comment]
Nice! Will have a look probably end of next week, thanks for the hint!

Resolution for Google play could be having it released by ownCloud, I do have a Dev account so I could also release it, if that is the only trouble in getting it out there ;)

**Comment by stefan-niedermann** [comment]
Hey guys, thank you all very much for your interest :)
@AndyScherzinger would be great if you could help me with F-Droid Publish. I already tried it, my results are here: https://f-droid.org/forums/topic/owncloud-notes/

> Resolution for Google play could be having it released by ownCloud, I do have a Dev account so I could also release it, if that is the only trouble in getting it out there ;)

I wanted to sell it on GPlay and provide it for free on F-Droid, but i don't have a credit card ;) so i'm fine if you want to publish it on GPlay ;)

I would be happy for everyone who likes to review the code and creates patches, makes suggestions and helps me - thats my first native android app, so the project may contain some horrible code snippets :laughing: 

regards

**Comment by AndyScherzinger** [comment]
I don't know much about the f-droid workflows :(

It seems one needs to have the following part in the post, e.g.: 
Source Code:https://github.com/PedroCarrillo/Expense-Tracker-App
License:https://github.com/PedroCarrillo/Expense-Tracker-App/blob/master/license.txt
Issue Tracker:https://github.com/PedroCarrillo/Expense-Tracker-App/issues
Readme:https://github.com/PedroCarrillo/Expense-Tracker-App/blob/master/README.md

**Comment by stefan-niedermann** [comment]
F-Droid-Publish moved to https://github.com/stefan-niedermann/OwnCloud-Notes/issues/13 because it's off-topic here and i don't want to blow the server issuetracker.

@jancborchardt @David-Development @davivel - who ever can help :)

**Comment by jancborchardt** [comment]
> I wanted to sell it on GPlay and provide it for free on F-Droid, but i don't have a credit card ;)

You’re in Germany, right? Nearly every bank issues credit cards for free or cheap (depending on your age). Other than that I heard good things from friends about DKB.
Maybe you don’t even need a credit card, just a Google Wallet account with a bank account or PayPal connected?

Would be really cool if you can publish it through your own account on Google Play and make some money off the great free software you code. :)

**Comment by stefan-niedermann** [comment]
ownCloud Notes has been released on F-Droid and Google Play Store.
If you want, you can adjust your Wiki-Page.

The links are mentioned in the README.md at https://github.com/stefan-niedermann/OwnCloud-Notes

I would also remove the "beta"-hint. We have fixed many bugs in the last weeks and i would mark the app as stable.

**Comment by Henni** [comment]
@stefan-niedermann done: https://github.com/owncloud/notes/wiki

Maybe we should even highlight clients in the readme
@jancborchardt what do you think? Feel free to edit the readme :wink:

**Comment by jancborchardt** [comment]
I moved MyOwnNotes down because it currently is not maintained, unfortunately. :(
